gradle/libs.versions.toml

```diff
@@ -21,7 +21,7 @@ protobufPlugin = "0.9.4"
 androidxActivityCompose = "1.8.2"
 androidxAppCompat = "1.6.1"
 androidxBenchmark = "1.2.3"
-androidxCamera = "1.5.0-SNAPSHOT"
+androidxCamera = "1.5.0-alpha02"
```

Comment:
This should stay as a `SNAPSHOT`. If you need a newer API, then update the build ID in settings.gradle.